cloudflare_rules/cloudflare_firewall_ddos.py

```diff
@@ -0,0 +1,8 @@
+def rule(event):
+    if event.get("Source") != "l7ddos":
+        return False
+    return True
+
+
+def title(_):
+    return "Cloudflare Detected L7 DDoS"
```

Comment:
Thoughts on appending the ClientRequestHost?

Reply:
If we were to append `ClientRequestHost` we would also want to dedup off of that key, however since this DDoS vs a DoS, that would seem to imply coming from multiple IPs vs a single IP